src/GraphQL.Conventions/Adapters/ResolutionContext.cs

```diff
@@ -23,9 +23,8 @@ public object GetArgument(string name, object defaultValue = null)
         {
             lock (_lock)
             {
-                object value;
                 if (FieldContext.Arguments != null &&
-                    FieldContext.Arguments.TryGetValue(name, out value))
+                    FieldContext.Arguments.TryGetValue(name, out var value))
```

Comment:
For readability, I would be specific about the type (here and other changes in this PR), even if it's object or TValue.